draft-bbs-signatures.md

```diff
@@ -534,9 +532,9 @@ Procedure:
 
 1. signature_result = octets_to_signature(signature)
 
-2. (i1, i2,..., iR) = RevealedIndexes
+2. [(H_i1, msg_i1), ..., (H_iR, msg_iR)] = [(H_i, msg_i), if revealed_i = 1]
```

Comment:
I think this syntax needs to be explained more, in general we could elect to do more of a filter or map style function?

Reply:
I think that the complexity of having to define a filter function will outweigh the advantages of the current API??? In my last push I've described in "clear text" the operation of getting `(H_i1, msg_i1), ..., (H_iR, msg_iR)` etc., Thoughts on the alternative???